lib/migration.js

```diff
@@ -479,9 +479,9 @@ function mixinMigration(PostgreSQL) {
     const p = propertyDefinition;
     switch (p.type.name) {
       default:
-      case 'String':
       case 'JSON':
-        return 'TEXT';
+        return 'JSONB';
```

Comment:
What is the impact of this change on existing application? Have you considered introducing a new `p.type.name` called `JSONB`, that will allow app developers using `JSON` to preserve the current behavior, and let them opt into the new JSONB-based storage?